tests/helpers/graphical_units.py

```diff
@@ -48,14 +50,13 @@ def set_test_scene(scene_object, module_name):
         data == np.array([0, 0, 0, 255]),
     ), f"Control data generated for {str(scene)} only contains empty pixels."
     assert data.shape == (480, 854, 4)
-    tests_directory = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    path_control_data = os.path.join(
-        tests_directory,
+    tests_directory = Path(__file__).absolute().parent.parent
+    path_control_data = Path(tests_directory).joinpath(
         "control_data",
         "graphical_units_data",
     )
-    path = os.path.join(path_control_data, module_name)
-    if not os.path.isdir(path):
-        os.makedirs(path)
-    np.savez_compressed(os.path.join(path, str(scene)), frame_data=data)
+    path = Path(path_control_data).joinpath(module_name)
+    if not path.is_dir():
+        path.mkdir(parents=True)
+    np.savez_compressed(Path(path).joinpath(str(scene)), frame_data=data)
```

Comment:
As path is already a pathlib object, you don't have to type Path(path), but you can use path directly.

Reply:
Resolved!